site/content/user-guide/device-registry.md

```diff
@@ -127,6 +127,30 @@ The following table provides an overview of the configuration variables and corr
 | `HONO_AUTH_VALIDATION_CERT_PATH`<br>`--hono.auth.validation.certPath` | no  | - | The absolute path to the PEM file containing the public key that the service should use for validating tokens issued by the Authentication service. Alternatively, a symmetric key can be used for validating tokens by setting the `HONO_AUTH_VALIDATION_SHARED_SECRET` variable. If none of these variables is set, the service falls back to the key indicated by the `HONO_AUTH_CERT_PATH` variable. If that variable is also not set, startup of the service fails. |
 | `HONO_AUTH_VALIDATION_SHARED_SECRET`<br>`--hono.auth.validation.sharedSecret` | no  | - | A string to derive a symmetric key from which is used for validating tokens issued by the Authentication service. The key is derived from the string by using the bytes of the String's UTF8 encoding. When setting the validation key using this variable, the Authentication service **must** be configured with the same key. Alternatively, an asymmetric key pair can be used for validating (and signing) by setting the `HONO_AUTH_SIGNING_CERT_PATH` variable. If none of these variables is set, startup of the service fails. |
 
+## Device Identities File Format
+
+The Device Registry supports persisting the device identities and their registration information to a JSON file in the local file system.
+The *Getting started Guide* includes an example configuration which illustrates the file format used. The configuration file's location is `/example/src/main/config/example-device-identities.json`.
+
+## Credentials File Format
+
+The Device Registry supports persisting the devices' credentials to a JSON file in the local file system.
+The *Getting started Guide* includes an example configuration which illustrates the file format used. The configuration file's location is `/example/src/main/config/example-credentials.json`.
+
+## Configuring Gateway Devices
+
+The Device Registry supports devices to *act on behalf of* other devices. This is particularly useful for cases where a device does not connect directly to a Hono protocol adapter but is connected to a *gateway* component that is usually specific to the device's communication protocol. It is the gateway component which then connects to a Hono protocol adapter and publishes data on behalf of the device(s). Examples of such a set up include devices using *SigFox* or *LoRa* for communication.
+
+In these cases the protocol adapter will usually authenticate the gateway component instead of the device for which it wants to publish data. In order to verify that the gateway is *authorized* to publish data on behalf of the particular device, the protocol adapter can include the gateway's device identifier (as determined during the authentication process) in its invocation of the Device Registry's *assert Device Registration* operation.
```

Comment:
, the protocol adapter **must** include the gateway's device...

IMHO the stronger phrase covers better what is meant here. WDYT?

Reply:
since we cannot enforce it, and this is not a specification, we might want to use *should*?

Reply:
*should* :+1: